pkg/model/components/context.go

```diff
@@ -72,7 +76,7 @@ func UsesKubenet(clusterSpec *kops.ClusterSpec) (bool, error) {
 	} else if networking.External != nil {
 		// external is based on kubenet
 		return true, nil
-	} else if networking.CNI != nil || networking.Weave != nil || networking.Flannel != nil || networking.Calico != nil || networking.Canal != nil || networking.Kuberouter != nil {
```

Comment:
This change seems unrelated to this PR,  should remove it to avoid any confusion in the future